x/outline-connectivity-app/.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "typescript.tsdk": ".yarn/sdks/typescript/lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true,
+  "search.exclude": {
```

Comment:
I have the impression vscode excludes what's in .gitignore, so we may not need this.